py3status/modules/file_status.py

```diff
@@ -68,12 +65,18 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
-            color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+        paths = glob(expanduser(self.path))
+        if isinstance(paths, str):
+            paths = [paths]
```

Comment:
When we `glob` stuffs, is `paths` not always a list? Also, to get rid of `expanduser` here.

Reply:
Yes, it is not always a list, if there is nothing to glob the return stay a str

Reply:
I don't know what to make of this. Consult with the fam.

Reply:
weird, will retry
```
glob.glob(pathname, *, recursive=False)
Return a *possibly-empty list* of path names that match pathname, which must be a string containing a path specification. pathname can be either absolute (like /usr/src/Python-1.5/Makefile) or relative (like ../../Tools/*/*.gif), and can contain shell-style wildcards. Broken symlinks are included in the results (as in the shell).
```